packages/trader/src/AppV2/Hooks/useActiveSymbols.ts

```diff
@@ -42,20 +50,20 @@ const useActiveSymbols = ({ barrier_category = [] }: TUseActiveSymbols) => {
                 response = await WS.activeSymbols(request);
             }
 
-            const { active_symbols, error } = response;
-
-            setActiveSymbolsV2(active_symbols);
-
-            if (!active_symbols?.length || error) {
+            const { active_symbols = [], error } = response;
+            if (error) {
+                showError({ message: localize('Trading is unavailable at this time.') });
```

Comment:
NOTE 1. I'm using the logic from trade-store: in case of an error, we call showError() without setting active_symbols.
NOTE 2. showError does not accept big Localize. I will not touch it to make the message accept React.ReactNode because this method is from common-store and it's used in other packages.